code/game/objects/structures/crates_lockers/closets.dm

```diff
@@ -369,7 +368,7 @@
 				close()
 	else
 		O.forceMove(T)
-	return 1
+	return TRUE
```

Comment:
Unrelated changes

Reply:
I'm touching this file anyways, why not.

Reply:
Makes merge conflicts more likely, makes reviewing harder since rather than focusing on 1 section of the code it interlinks with more random sections of the code which have no correlation and in turn causes reviews to be slower and the PRs to be way more likely to not get reviewed and just expire

Reply:
Really gotta think hard about how this change affects other parts of the code, yes...